api/src/nv_ingest_api/internal/extract/image/chart_extractor.py

```diff
@@ -27,7 +27,7 @@
 PADDLE_MIN_WIDTH = 32
 PADDLE_MIN_HEIGHT = 32
 
-logger = logging.getLogger(f"morpheus.{__name__}")
+logger = logging.getLogger(f"ray.{__name__}")
```

Comment:
I think this snuck in.

```suggestion
logger = logging.getLogger(f"{__name__}")
```